File: examples-sdk/zig/507-villog-bench-file-drain/main.zig
// 507-villog-bench-file-drain — Zig SDK equivalent
// Compile: vil compile --from zig --input 507-villog-bench-file-drain/main.zig --release

const vil = @import("vil");

pub fn main() void {
    var server = vil.Server.init("app", 8080);
    server.compile();
}

File: examples-sdk/go/507-villog-bench-file-drain/main.go
// 507-villog-bench-file-drain — Go SDK equivalent
// Compile: vil compile --from go --input 507-villog-bench-file-drain/main.go --release
package main

import vil "github.com/OceanOS-id/vil-go"

func main() {
	s := vil.NewServer("app", 8080)
	s.Compile()
}
